pkg/controller/rbac.go

```diff
@@ -65,6 +69,75 @@ func (c *StashController) ensureSidecarRoleBindingDeleted(resource metav1.Object
 		Delete(c.getSidecarRoleBindingName(resource.Name), &metav1.DeleteOptions{})
 }
 
+func (c *StashController) ensureCronJobClusterRole() error {
+	meta := metav1.ObjectMeta{
+		Name: CronJobClusterRole,
+	}
+	_, _, err := rbac_util.CreateOrPatchClusterRole(c.kubeClient, meta, func(in *rbac.ClusterRole) *rbac.ClusterRole {
+		deployment, err := c.kubeClient.AppsV1().Deployments(os.Getenv(util.OperatorName)).Get(os.Getenv(util.OperatorNamespace), metav1.GetOptions{})
```

Comment:
Run operator inside cluster to see if it works.

Reply:
fixed